db/migrate/20201027221804_install_audited.rb

```diff
@@ -0,0 +1,30 @@
+class InstallAudited < ActiveRecord::Migration[6.0]
+  def self.up
+    create_table :audits, :force => true do |t|
```

Comment:
Out of curiosity - @holmesie why the explicit force here? I see we haven't done it for table creations in previous migrations.

Reply:
This is boilerplate from the audit gem; force just drops this table if it already exists (for some reason).

Reply:
Ah okay, was wondering if it was just the default functionality of that gem. Sounds good!